### PR TITLE
[Core] Fix bug where $val=0 would pass as blank!

### PR DESCRIPTION
**Pull request by gluneau**
This bug affected the upload capability of LorisForm for addFile()

cause:
```{r_tidy=false}
    [recording_file] => Array
        (
            [name] => Screenshot from 2016-08-09 18:54:44.png
            [type] => image/png
            [tmp_name] => /tmp/phpNvwMAZ
            [error] => 0
            [size] => 261121
        )
```

### CODE REVIEW

File: php/libraries/NDB_BVL_Instrument.class.inc [hunk 1]
```diff
@@ -1080,7 +1080,7 @@ class NDB_BVL_Instrument extends NDB_Page
             $flag = true;
             if (is_array($elvalue)) {
                 foreach ($elvalue AS $val) {
-                    if ($val == "") {
+                    if ($val === "") {
```
Comment (ridz1208) [inline php/libraries/NDB_BVL_Instrument.class.inc]: @gluneau  This change should also be done on line 1091 as the same comparison is made

**Comment by ridz1208** [comment]
@gluneau even tho this does fix the issue, it seems to be a change in behaviour. some code might be relying on it (http://php.net/manual/en/types.comparisons.php)

File: php/libraries/NDB_BVL_Instrument.class.inc (hunk 1 above)
Comment (ridz1208) [inline php/libraries/NDB_BVL_Instrument.class.inc]: After a quick check across instruments, it seems like they would have similar returns for other elements

**Comment by ridz1208** [comment]
After a quick check across instruments, it seems like they would have similar returns for other elements

**Comment by driusan** [comment]
@ridz1208 This is still tagged as requested changes by you. It looks fine to me, is there anything wrong with it?

**Pull request merged**